k8s/prod/common/rse/check.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name:rse-check
+  namespace: check
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: rse-check
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rse-check
+    version: 0.0.7
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rse/check:latest
+      ingressHost: rse-check-prod.service.core-compute-prod.internal
```

Comment:
Ingress can move to base chart as templatedgenerally

Reply:
I have remove the prod config